zipline/assets/roll_finder.py

```diff
@@ -149,7 +147,15 @@ def __init__(self, trading_calendar, asset_finder, session_reader):
         self.session_reader = session_reader
 
     def _active_contract(self, oc, front, back, dt):
-        # FIXME: Possible vector for look ahead bias.
-        front_vol = self.session_reader.get_value(front, dt, 'volume')
-        back_vol = self.session_reader.get_value(back, dt, 'volume')
-        return back if back_vol > front_vol else front
+        for i, sid in enumerate(oc.contract_sids):
+            if sid == front:
+                break
```

Comment:
Also, have been considering changing the implementation so that we are using a contract object which has some structure like:

```
class ChainedContract(object):
    def. __init__(...):
        self.future = ... # A Future object
        self.next = # Next contract in the chain
        self.prev = # Prev contract in the chain
```

Where `chained_contarct.future.sid` would be the way to get the current sid. 
But would do that after merging this.